tree/tree/src/TBranch.cxx

```diff
@@ -1279,6 +1279,14 @@ Int_t TBranch::GetEntry(Long64_t entry, Int_t getall)
             fNextBasketEntry = -1;
             return -1;
          }
+         if (fTree->GetMaxVirtualSize() < 0) {
```

Comment:
Is `fBits` really the right direction to go here (i.e., do we have enough bits left -- what are the backward compatibility concerns)?